relay/workflow.py

```diff
@@ -1832,7 +1832,7 @@ async def resolve_incident(self, incident_id: str, reason: str):
     #         '_target': self.targets_from_source_uri(target),
     #         'restart': True
     #     }
-    #     await self.sendReceive(event)
+    #     await self._sendReceive(event)
```

Comment:
I know restart_device and power_down_device are commented out for now, but just noting that there's a typo for these when calling _send_receive(event)

Reply:
Good eyes!! I just pushed another commit to address that.